articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -168,12 +171,11 @@ namespace TestConsole
 }
 ```
 
-The [SetDirty](/dotnet/api/microsoft.extensions.configuration.azureappconfiguration.iconfigurationrefresher.setdirty) method is used to set the cached value for key-values registered for refresh as dirty. This ensures that the next call to `RefreshAsync` or `TryRefreshAsync` re-validates the cached values with App Configuration and updates them if needed.
+The `ProcessPushNotification` method resets the cache expiration to a short random delay. This causes future calls to `RefreshAsync` or `TryRefreshAsync` to re-validate the cached values against App Configuration and update them as necessary. In this example you register to monitor changes to the key: *TestApp:Settings:Message* with a cache expiration of one day. This means no request to App Configuration will be made before a day has passed since the last check. By calling  `ProcessPushNotification` your application will send requests to App Configuration in the next few seconds. Loading new configuration values shortly after changes occur in App Configuration without constantly polling for updates. In case your application misses the change notification for any reason, it will still check for configuration changes once a day.
```

Comment:
> Loading new configuration values shortly after changes occur in App Configuration without constantly polling for updates.

This sentence doesnt make sense.

Reply:
Does this sound better to you?

"Your application will load the new configuration values shortly after changes occur in the App Configuration store without the need to constantly poll for updates."

Reply:
Yes, sounds good.